kubernetes/configs/test/authorization_management.sh

```diff
@@ -2,3 +2,5 @@
 
 AUTHORIZATION_MANAGEMENT_RESOURCE_CPU="4"
 AUTHORIZATION_MANAGEMENT_RESOURCE_MEM="8Gi"
+
+ENABLED_PROJECTIONS="false"
```

Comment:
⚠️ despite being defined here, this value is still global, so it will be valid for all services that uses this var, like the tenant management.
If you want to apply it only for the auth management it should be overridden in the manifest with kustomize.

Since the projection code has been removed, is this necessary?